package.json

```
@@ -3,10 +3,22 @@
   "description": "An Esprima-compatible JavaScript parser built on Acorn",
   "author": "Nicholas C. Zakas <nicholas+npm@nczconsulting.com>",
   "homepage": "https://github.com/eslint/espree",
-  "main": "espree.js",
+  "main": "dist/espree.cjs",
+  "type": "module",
+  "exports": {
+    ".": [
+      {
+        "import": "espree.js",
+        "default": "./dist/espree.cjs"
+      },
+      "dist/espree.cjs"
+    ],
+    "./package.json": "./package.json"
+  },
```

```suggestion
  "exports": {
    ".": [
      {
        "import": "./espree.js",
        "default": "./dist/espree.cjs"
      },
      "./dist/espree.cjs"
    ],
    "./package.json": "./package.json"
  },
```

I think that these paths must start with `./`.